Fix don't usage PHP attributes in `RulesNormalizer::normalize()` + make `$rules` optional (#733)
yiisoft/validator: CHANGELOG.md

  (@arogachev)
- Enh #726: Refactor `Result::add()`: took `array_merge()` out of the `foreach` (@lav45)
- Chg #583: Change "attribute" to "property" in class/trait/method/variable/placeholder names (@vjik)
- Enh #733: Make parameter `$rules` in `RulesNormalizer::normalize()` optional (@vjik)

> - Bug #733: Fix don't usage rules provided by data object PHP attributes in `RulesNormalizer::normalize()` (@vjik)

Reviewer: ```suggestion
- Bug #733: Rules provided by data object PHP attributes were not used in  `RulesNormalizer::normalize()` (@vjik)
```
